kamon-core/src/main/resources/reference.conf

```diff
@@ -142,7 +144,7 @@ kamon {
     }
 
     # Default dispatcher for all trace module operations
-    dispatcher = ${kamon.default-dispatcher}
+    dispatcher = "akka.actor.default-dispatcher"
```

Comment:
Right, I think we already discussed that before and I just forgot.. will introduce a small dispatcher config just for Kamon.